clmm/utils/boost.py

```diff
@@ -1,24 +1,39 @@
 """General utility functions that are used in multiple modules"""
+
 import numpy as np
 
 
-def compute_nfw_boost(rvals, rscale=1000, boost0=0.1):
-    """Given a list of `rvals`, and optional `rscale` and `boost0`, return the corresponding
-    boost factor at each rval
+def compute_nfw_boost(rvals, rscale, boost0=0.1):
+    r"""Computes the boost factor at radii rvals using a parametric form 
+    following that of the analytical NFW excess surface density.
+
+    Setting :math:`x = R/R_s` 
```

Comment:
I think it will be clearer to write `x = rvals/rscale`, so poeple know where the arguments are in the equation. We can also rename the argument to `r_proj`, so it is consistent with the theory module.

Reply:
Just saw that there was an extra space at the end of this line.